pandas/tests/util/test_assert_produces_warning.py

```diff
@@ -0,0 +1,17 @@
+import warnings
+
+import pytest
+
+import pandas.util.testing as tm
+
+
+def f(a=FutureWarning, b=RuntimeWarning):
+    warnings.warn('f1', a)
+    warnings.warn('f2', b)
+
+
+@pytest.mark.filterwarnings('ignore:f1:FutureWarning')
+@pytest.mark.filterwarnings('ignore:f2:RuntimeWarning')
+def test_assert_produces_warning_honors_filter():
+    with tm.assert_produces_warning(RuntimeWarning):
```

Comment:
Shouldn't you be testing the new option here?

Reply:
Yes... I've messed something up, sorry.